src/server.ts

```diff
@@ -17,17 +17,11 @@ export interface RequestBody<A> {
   value: A;
 }
 
-export interface Params {
-  [param: string]: unknown;
-}
+export type Params = object;
```

Comment:
This is pretty much the only change necessary.
TS 3.6 complains unless every object used as `Params` also have `[param: string]: unknown;`, and most of them don't since they have pre-defined fields instead.
Assuming your intention with that was to enforce `Params` (and `Headers`, and `Query`) were always an object, that behaviour should be maintained after this change

Reply:
that might be ok, there was something fairly intricate about having unknown valued keys but if this works for us then it might be ok